src/pages/Cart.jsx

```diff
@@ -1,24 +1,39 @@
-import {
-  faSearch,
-  faTrash,
-  faCartShopping,
-} from "@fortawesome/free-solid-svg-icons";
+import { faSearch, faCartShopping } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { useContext, useState } from "react";
 import { CartContext } from "../context/CartProvider";
 import ItemsCart from "../components/ItemsCart";
 
 const Cart = () => {
-  const { cartItems, cartTotal } = useContext(CartContext);
+  const { cartItems, cartTotal, cartCheckout } = useContext(CartContext);
   const [searchTerm, setSearchTerm] = useState("");
+  const [sortOrder, setSortOrder] = useState("");
+  const handleSort = (order) => {
+    setSortOrder(order);
+  };
 
-  const filteredData = cartItems.filter((item) => {
-    if (searchTerm === "") {
-      return item;
-    } else {
-      return item.product.title.toLowerCase().includes(searchTerm);
-    }
-  });
+  const filteredData = cartItems
```

Comment:
Esto podría moverse a una función en un archivo externo

```suggestion
  const filteredData = filterData(cartItems);
  
  // utils.js
  const filterData = (items) => {
   ...
  }
```